CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v0.71 [2023-06-21]
+
+_Enhancements_
+
+- Removed the following controls from `foundational_security` benchmark to better align with the matching [AWS Security Hub](https://docs.aws.amazon.com/securityhub/latest/userguide/doc-history.html):
+  - `foundational_security_ec2_27`
+  - `foundational_security_lambda_4`
```

Comment:
@madhushreeray30 could you please update the section per https://github.com/turbot/steampipe-mod-aws-compliance/blob/2a6912b7c0d365a854001c1b087212e18846ef58/CHANGELOG.md#v063-2023-04-20